trace_point.rb

```diff
@@ -168,14 +168,15 @@ def self.allow_reentry
   #                   #   trace is still enabled
   #
   # If a block is given, the trace will only be enabled within the scope of the
```

Comment:
```suggestion
  # If a block is given, the trace will only be enabled during the block call.
```
That seems much clearer than `scope` which might sound lexical but is not the case.